kubeflow/mnist/k8s/mpi_job_mnist.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: "kubeflow.org/v2beta1"
+kind: "MPIJob"
+metadata:
+  name: "pytorch-horovod-mnist-mpi"
+spec:
+  slotsPerWorker: 4
+  runPolicy:
+    cleanPodPolicy: Running
+  mpiReplicaSpecs:
+    Launcher:
+      replicas: 1
+      restartPolicy: OnFailure
+      template:
+        spec:
+          containers:
+            - name: pytorch
+              image: navarrepratt/pytorch_mpi_mnist:1
+              command:
+                - "mpirun"
+                - "-n"
+                - "4"  # Total processes = num workers * slots per workers
+                - "--allow-run-as-root"
+                - "python"
+                - "/opt/mnist/src/mnist.py"
+                - "--data-dir"
+                - "/mnt/pvc/dataset"
+                - "--model-dir"
+                - "/mnt/pvc/mpi/checkpoints"
+              resources:
```

Comment:
You want an affinity to schedule on a CPU. node in the same region as the worker (LAS1)